assets/styles/base.css

```diff
@@ -0,0 +1,309 @@
+@font-face {
+  font-family: bold;
+  src: url(../fonts/bold.woff2);
+}
+
+@font-face {
+  font-family: regular;
+  src: url(../fonts/regular.woff2);
+}
+
+:root {
+  --accent-color: #ed5342;
+  --light-color: #d3d3d3;
+  --dark-color: #272a31;
+  --bright-color: #fff;
+  --text-color: #666;
+  --title-font: bold, serif;
+  --text-font: regular, serif;
+  --icons-font: icons, serif;
+}
+
+html,
+body {
+  padding: 0;
+  margin: 0;
+  color: var(--text-color);
+  font: 12px var(--text-font);
+}
+
+h1,
+h2,
+h3,
+ul,
+p,
+hr,
+button {
+  padding: 0;
+  margin: 0;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+  transition: all 0.1s ease;
+}
+
+button {
+  color: inherit;
+  background-color: transparent;
+  border: 0;
+}
+
+.container {
+  padding: 0 10px;
+}
+
+.line {
+  width: 50px;
+  border: 1px solid var(--accent-color);
+  margin: 10px auto;
+}
+
+.visibility {
+  display: flex !important;
+}
+
+.scrolling {
+  overflow: hidden;
+}
+
+.accent-color {
+  color: var(--accent-color);
+}
+
+.light-color {
+  color: var(--light-color);
+}
+
+.dark-color {
+  color: var(--dark-color);
+}
+
+.bright-color {
+  color: var(--bright-color);
+}
+
+.accent-background {
+  background-color: var(--accent-color);
+}
+
+.light-background {
+  background-color: var(--light-color);
+}
+
+.dark-background {
+  background-color: var(--dark-color);
+}
+
+.bright-background {
+  background-color: var(--bright-color);
+}
+
+.header {
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  inset-inline: 0;
+}
+
+.header-menu {
+  width: 50px;
+  height: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 2.25rem;
+}
+
+.header nav {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  inset-inline: 0;
+  display: none;
+  flex-direction: column;
+  color: var(--bright-color);
+  background-color: var(--dark-color);
+}
+
+.header-close {
+  width: 50px;
+  height: 50px;
+  text-align: center;
+  font-size: 2.25rem;
+  align-self: flex-end;
+  margin-top: 10px;
+  margin-inline-end: 10px;
+}
+
+.header-links {
+  flex-direction: column;
+  font: 2.25rem var(--title-font);
+  margin: 15px 0;
+}
+
+.header-links li {
+  padding: 10px 0;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.05);
+}
+
+.header-logo {
+  display: none;
+}
+
+.header-button {
+  width: 200px;
+  display: block;
+  font-size: 1.5rem;
+  padding: 10px;
+  text-align: center;
+  border: 5px solid var(--accent-color);
+  margin: auto;
+}
+
+.header-social {
+  gap: 20px;
+  font-size: 1.75rem;
+  justify-content: center;
+}
+
+.header-social li {
+  width: 40px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.footer {
+  padding: 50px 0;
+}
```

Comment:
- [ ] Good job on the Footer section, however, please ensure that your alignment matches the given design. a possible solution is to give `display:flex` properties and adjust them according to the design requirement. 

Below is an illustrations of yours and the given footer Design: :arrow_down: 

<table>
    <tr>
      <td>Your Footer Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student design footer](https://github.com/mahammad-mostafa/game-industry-conference-site/assets/43528594/d7b3aee4-9d3a-4085-8ac7-2fa932fcb7f7)


</td>
      <td>

![design footer](https://github.com/mahammad-mostafa/game-industry-conference-site/assets/43528594/3433e451-ef5a-4030-8e20-b220b8f32b97)


</td>
    </tr>
  </table>

<table>
    <tr>
      <td>Your About Footer Design</td>
      <td>Given About Footer Design</td>
    </tr>
    <tr>
      <td>

![Student about Footer](https://github.com/mahammad-mostafa/game-industry-conference-site/assets/43528594/1ae88006-fa4a-4861-8190-fcd4f1c149de)

</td>
      <td>

![Design about Footer](https://github.com/mahammad-mostafa/game-industry-conference-site/assets/43528594/93a27369-a680-49ea-a769-b8eaad232f6c)


</td>
    </tr>
  </table>